tests/02_00_05-2wp_version1.js

```diff
@@ -344,7 +154,7 @@ describe('Lock funds using peg-in protocol version 1', () => {
 
         expect(rejectedPeginEvent).to.not.be.null;
         expect(rejectedPeginEvent.arguments.btcTxHash).to.equal(peginBtcTxHashWith0xPrefix);
-        expect(rejectedPeginEvent.arguments.reason).to.equal(PEGIN_V1_INVALID_PAYLOAD_REASON);
+        expect(rejectedPeginEvent.arguments.reason).to.equal(LEGACY_PEGIN_MULTISIG_SENDER_REASON);
```

Comment:
Isn't this what Natha fixed? That rejected reason doesn't seem right

Reply:
Updated!